### ISSUE TITLE
Doesn't work with jQuery 3.1.1

### ISSUE THREAD

**Issue by DedicatedManager** [saribe/eModal#62]
If I use jquery 3.1.1 it doesn't work.  I get javascipt/jquery errors.  If I use 2.2, it works.  Will this be updated for the latest version of jquery?
Thanks,
tc

Doesn't work:
`<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>`

Works:
` <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>`

**Comment by DevStrefa** [comment]
I was able to run this library with jquery 3.1.1 with some changes in emodal.js

`
var message = $(html)
                .load(iframeReady);
`
change to 
`
var message=$(html).on('load', iframeReady);
`

**Comment by thegde** [comment]
I have added jQuery Migrate to make it work

**Comment by DedicatedManager** [comment]
Thanks for the response...

So you are saying you got it to work with jQuery like this?

<script src="https://code.jquery.com/jquery-3.0.0.js"></script>
<script src="https://code.jquery.com/jquery-migrate-3.0.0.js"></script>

Taken from the documentation at:
https://github.com/jquery/jquery-migrate

**Comment by thegde** [comment]
Yes It worked for me with this.. I did not have to make any other changes
